lbjava/src/test/java/edu/illinois/cs/cogcomp/lbjava/algo_test/main/AdaGradCLMain.java

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
We will unify the AdaGrad and SGD, with classification and regression versions. I will work on this once MLP works.